homeassistant/components/ezviz/config_flow.py

```diff
@@ -303,11 +310,76 @@ async def async_step_confirm(self, user_input=None):
             data_schema=discovered_camera_schema,
             errors=errors,
             description_placeholders={
-                "serial": self.unique_id,
+                ATTR_SERIAL: self.unique_id,
                 CONF_IP_ADDRESS: self.context["data"][CONF_IP_ADDRESS],
             },
         )
 
+    async def async_step_reauth(self, user_input: Mapping[str, Any]) -> FlowResult:
+        """Handle a flow for reauthentication with password."""
+
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a Confirm flow for reauthentication with password."""
+        auth_data = {}
+        errors = {}
+        entry = None
+
+        for item in self._async_current_entries():
+            if item.data.get(CONF_TYPE) == ATTR_TYPE_CLOUD:
+                self.context["title_placeholders"] = {ATTR_SERIAL: item.title}
+                entry = await self.async_set_unique_id(item.title)
```

Comment:
We shouldn't rely on the config entry title for any logic or api calls. The user can change the title.

If you need the serial number it should be stored as config entry data.

Reply:
Addressed in https://github.com/home-assistant/core/pull/94384